engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/SimpleQueryStringPredicateTerminalContext.java

```diff
@@ -26,6 +26,20 @@
 	 */
 	SimpleQueryStringPredicateTerminalContext withAndAsDefaultOperator();
 
-	// TODO HSEARCH-3312 allow analyzer/normalizer override
+	/**
+	 * Define an analyzer to use at query time to interpret the value to match.
+	 * Otherwise, it will be used, if present, the one defined on the field.
```

Comment:
This would be the grammatically correct way to express what you meant:

```
<p>
If this method is not called, the analyzer defined on the field will be used.
```